### PR TITLE
fix terserutil clear

### PR DESCRIPTION
**Pull request by fil512**

### CODE REVIEW

File: hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java [hunk 1]
```diff
@@ -570,6 +572,26 @@ public void testClearFieldByFhirPath() {
 		assertEquals("Test", p1.getAddress().get(1).getCity());
 	}
 
+	@Test
+	void testRemoveByFhirPath() {
+		// arrange
+		FhirContext fhirContext = FhirContext.forR4();
```
Comment (AD1306) [inline hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java:578]: you can use `ourFhirContext` instead.

File: hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java [hunk 2]
```diff
@@ -570,6 +572,26 @@ public void testClearFieldByFhirPath() {
...
+		claimWithReferences.setPatient(new Reference("Patient/123"));
+		String fhirPath = "patient";
+		assertTrue(claimWithReferences.hasPatient());
+		//act
+		TerserUtil.clearFieldByFhirPath(fhirContext, claimWithReferences, fhirPath);
+		//assert
+		assertFalse(claimWithReferences.hasPatient());
+	}
+
+	public static Claim createClaim() {
```
Comment (AD1306) [inline hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java:589]: nitpick: does this needs to be public ?

**Review by AD1306 — APPROVED** [review]

File: hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java (hunk 1 above)
Comment (fil512) [reply hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java]: fixed

File: hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java (hunk 2 above)
Comment (fil512) [reply hapi-fhir-structures-r4/src/test/java/ca/uhn/fhir/util/TerserUtilTest.java]: fixed

**Pull request merged**